backend/models/user.py

```diff
@@ -7,10 +7,18 @@ class User(UserMixin, db.Model):
     id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(64), index=True,
                       unique=True, nullable=False)
-    photo = db.Column(db.String(256), index=True,
-                      unique=True, nullable=False)
-    name = db.Column(db.String(64), index=True,
-                      unique=True, nullable=False)
+    photo = db.Column(db.String(256), index=False,
```

Comment:
i c u have changed it here. I think if we're making changes to the same file, don't spread it out across commits cause that can get messy. I would squash this with the previous commit and keep each commit atomic